docker_sample/src/main.rs

```diff
@@ -0,0 +1,25 @@
+use actix_web::{web, App, HttpResponse, HttpServer, Responder};
+
+async fn index() -> impl Responder {
+    println!("GET: /");
+    HttpResponse::Ok().body("Hello world!")
+}
+
+async fn index2() -> impl Responder {
+    println!("GET: /again");
+    HttpResponse::Ok().body("Hello world again!")
+}
+
+#[actix_rt::main]
+async fn main() -> std::io::Result<()> {
+    println!("Starting actix-web server");
+
+    HttpServer::new(|| {
+        App::new()
+            .route("/", web::get().to(index))
+            .route("/again", web::get().to(index2))
```

Comment:
How about using the codegen macros here  with `.service` instead of `.route` directly? The [basics](https://github.com/actix/examples/blob/master/basics/src/main.rs#L17) example does it like that and I personally think this improves code readability, I think this is the default way of implementing routing since 1.0.